### ISSUE TITLE
Blank Pieces When Forking Repository

### ISSUE THREAD

**Issue by hank-w** [timburgan/timburgan#8670]
Hello, I was trying to setup a fork of this repository on my GitHub to try and do a player vs AI mode, but whenever I change the GitHub actions to be directed on my branch, the white pieces become blank on the board. Is there anyway to reset/start a completely new game, as I believe the issue is my branch not having previous history of moves, thus losing track of the pieces. Thank you!
@timburgan

**Comment by JRetza** [comment]
Try opening an issue titled "chess|new"

**Comment by timburgan** [comment]
You'll probably need to edit the workflow file too so it honors your own username to allow a force-restart of the game:

https://github.com/timburgan/timburgan/blob/master/.github/workflows/chess.yml#L66

**Comment by hank-w** [comment]
Thanks for the advice guys! I tried changing the workflow and repo in the chess.yml file to my forked repo, but each time it raises the issue in the original repo and also does not update my forked version. Are there other lines of code I need to configure to get it working? Thanks!

**Comment by timburgan** [comment]
> Are there other lines of code I need to configure to get it working? Thanks!

Yes probably. I did hardcode many things in the workflow file as this was just a quick experiment that I never imagined others would find interesting. There's API calls that call the Issues API and the Contents API that would all need to be changed to reference your own repo
